### PR TITLE
feat: Can go back to choose SDK page from show SDK instructions

### PR DESCRIPTION
**Pull request by dbolson**
A user can go back to choose another SDK once they're on the instructions page. We should show this in the help text once we have that.

**Requirements**

- [ ] I have added test coverage for new or changed functionality
- [ ] I have followed the repository's [pull request submission guidelines](../blob/main/CONTRIBUTING.md#submitting-pull-requests)
- [ ] I have validated my changes against all supported platform versions

**Related issues**

Provide links to any issues in this repository or elsewhere relating to this pull request.

**Describe the solution you've provided**

Provide a clear and concise description of what you expect to happen.

**Describe alternatives you've considered**

Provide a clear and concise description of any alternative solutions or features you've considered.

**Additional context**

Add any other context about the pull request here.

### CODE REVIEW

File: internal/quickstart/container.go [hunk 1]
```diff
@@ -19,17 +19,16 @@ const (
...
 	baseUri      string
 	currentModel tea.Model
 	currentStep  int
-	sdkKind      string
+	err          error
+	flagKey      string
+	flagsClient  flags.Client
+	quitMsg      string // TODO: set this?
+	quitting     bool
+	sdk          sdkDetail
```
Comment (dbolson) [inline internal/quickstart/container.go:31]: Added this and sorted the others.

File: internal/quickstart/container.go [hunk 2]
```diff
@@ -55,32 +54,59 @@ func (m ContainerModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
...
+		m.currentModel = NewShowSDKInstructionsModel(
+			m.accessToken,
+			m.baseUri,
+			msg.sdk.canonicalName,
+			msg.sdk.url,
+			m.flagKey,
+		)
 		cmd = m.currentModel.Init()
-		m.sdkKind = msg.sdkKind
+		m.sdk = msg.sdk
```
Comment (dbolson) [inline internal/quickstart/container.go:77]: I'm passing the entire sdk details instead of its fields.

File: internal/quickstart/container.go [hunk 3]
```diff
@@ -55,32 +54,59 @@ func (m ContainerModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
...
+		m.currentModel = NewToggleFlagModel(
+			m.flagsClient,
+			m.accessToken,
+			m.baseUri,
+			m.flagKey,
+			m.sdk.kind,
+		)
 		m.currentStep += 1
+	case selectedSDKMsg:
+		m.currentModel, cmd = m.currentModel.Update(msg)
```
Comment (k3llymariee) [inline internal/quickstart/container.go:107]: could this be grouped with the case statement for the msg's on line 95?

**Review by k3llymariee — APPROVED** [review]
just one comment about the key we're using but otherwise LGTM

File: internal/quickstart/container.go [hunk 4]
```diff
@@ -113,12 +139,17 @@ func (m ContainerModel) View() string {
...
 	Enter key.Binding
 	Quit  key.Binding
 	Tab   key.Binding
 }
 
 var keys = keyMap{
+	Back: key.NewBinding(
+		key.WithKeys("left"),
+		key.WithHelp("back", "go back"),
+	),
```
[the remark this answers is not in the document]
Comment (dbolson) [reply internal/quickstart/container.go]: Updated

File: internal/quickstart/container.go (hunk 3 above)
Comment (dbolson) [reply internal/quickstart/container.go]: Updated

**Pull request merged**